src/components/TownSquare.vue

```diff
@@ -292,6 +292,26 @@ export default {
             left: 100%;
           }
         }
+
+        .name {
+          flex-direction: row-reverse;
+
+          svg {
+            margin-left: 10px;
+            margin-right: 2px;
+          }
+
+          .pronouns {
+            left: 110%;
+            right: auto;
+            &:before {
+              border-left-color: transparent;
+              border-right-color: black;
+              left: auto;
+              right: 100%;
+            }
+          }
+        }
```

Comment:
```suggestion
        .name .pronouns {
          left: 110%;
          right: auto;
          &:before {
            border-left-color: transparent;
            border-right-color: black;
            left: auto;
            right: 100%;
          }
        }
```
The icon should always be in the same place - when things are positioned differently depending on the player seat, this usually happens for space reasons.